modules/logging-loki-pod-placement.adoc

```diff
@@ -0,0 +1,209 @@
+// Module included in the following assemblies:
+// logging/log_storage/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="logging-loki-pod-placement_{context}"]
+= Loki pod placement
+You can control which nodes the Loki pods run on, and prevent other workloads from using those nodes, by using tolerations or node selectors on the pods.
+
+You can apply tolerations to the log store pods with the LokiStack custom resource (CR) and apply taints to a node with the node specification. A taint on a node is a `key:value` pair that instructs the node to repel all pods that do not allow the taint. Using a specific `key:value` pair that is not on other pods ensures that only the log store pods can run on that node.
+
+.Example LokiStack with node selectors
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+# ...
+  template:
+    compactor: # <1>
+      nodeSelector:
+        node-role.kubernetes.io/infra: "" # <2>
+    distributor:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    gateway:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    indexGateway:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    ingester:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    querier:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    queryFrontend:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    ruler:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+# ...
+----
+
+<1> Specifies the component pod type that applies to the node selector.
+<2> Specifies the pods that are moved to nodes containing the defined label.
+
+In the previous example configuration, all Loki pods are moved to nodes containing the `node-role.kubernetes.io/infra: ""` label.
```

Comment:
I don't have a preference for this really, but would this maybe be better suited above the example, and then switching it to:

Just a thought

```suggestion
In the following example configuration, all Loki pods are moved to nodes containing the `node-role.kubernetes.io/infra: ""` label.
```